.github/workflows/bundle_cron.yml

```diff
@@ -0,0 +1,54 @@
+name: Update Bundles
+
+on:
+  schedule:
+    - cron: 0 5 * * *
+
+jobs:
+  check-repo-owner:
+    # This job is so the entire workflow will end successfully and give some
+    # output to explain why it hasn't run on a non-Adafruit fork.
+    runs-on: ubuntu-latest
+    steps:
+      - name: repository
+        env:
+          OWNER_IS_ADAFRUIT: ${{ startswith(github.repository, 'adafruit') }}
```

Comment:
Should this be `adafruit/` to make sure it isn't something like `adafruit-adabot/`? The other way to handle this would be to find the actual repo's owner data instead and check equality.

Reply:
Definitely a fair point. I clearly didn't even think about `adafruit-adabot` as a repo owner.